client/lclusterc.go

```diff
@@ -0,0 +1,316 @@
+/*
+ * File: client/main.go
+ *
+ * Description: Contains the lcluster client code.
+ *
+ * TODO: removejob is broken, others funcs are unimplemented
+ */
+
+package main
```

Comment:
Here and below, we follow

https://github.com/golang/go/wiki/CodeReviewComments

can you update the code accordingly?